src/components/structures/auth/Registration.tsx

```diff
@@ -223,7 +223,7 @@ export default class Registration extends React.Component<IProps, IState> {
                 this.setState({
                     flows: e.data.flows,
                 });
-            } else if (e.httpStatus === 403 && e.errcode === "M_UNKNOWN") {
+            } else if (e.httpStatus === 403 || e.errcode === "M_FORBIDDEN") {
```

Comment:
`||` because older Synapses used `M_UNKNOWN` wrongly

Reply:
Please add that as a code comment.